blog-generic/src/entities/common_minimal_author.rs

```diff
@@ -0,0 +1,31 @@
+use serde::{Deserialize, Serialize};
+use validator::{Validate, ValidationError};
+
+#[derive(Clone, Debug, Eq, PartialEq, Serialize, Deserialize, Validate)]
+#[serde(rename_all = "camelCase")]
+pub struct CommonMinimalAuthor {
```

Comment:
AuthorProfile, ProfileInfo maybe?

Reply:
trying to keep similar naming style with other structs